cli/Cargo.toml

```diff
@@ -8,6 +8,11 @@ edition = "2018"
 
 [dependencies]
 clap = "2.33.0"
+tar = "0.4"
+walkdir = "2"
+ignore = "0.4.7"
+tempfile = "3.0.8"
```

Comment:
The nice thing I learnt about this is the file is really removed once the program terminates